src/components/activity/activityStats/ActivityStatsAnalog.vue

```diff
@@ -109,24 +130,37 @@ export default Vue.extend({
       items: [],
       loading: false,
       search: '',
-      selectedRecords: null,
+      selectedRecord: null,
```

Comment:
How about the following?
```suggestion
      selectedRecord: undefined,
```